lib/request.js

```diff
@@ -31,7 +31,10 @@ module.exports = function generateExpressVirtualRequest(data, socket) {
     remoteAddress: '127.0.0.1'
   };
   req.protocol = 'http';
-  req.body = '';
+  req.secure = false;
+  req.xhr = false;
+  req.body = {};
```

Comment:
We should init `req.body` with `undfined`
see http://expressjs.com/api.html#req.body